src/pybind/mgr/dashboard/frontend/src/app/shared/components/config-option/config-option.component.html

```diff
@@ -0,0 +1,58 @@
+<div [formGroup]="optionsFormGroup">
+  <div *ngFor="let option of options; let last = last">
+    <div class="form-group"
+         [ngClass]="{'has-error': optionsForm.showError(option.name, optionsFormDir)}"
+         *ngIf="option.type === 'bool'">
+      <label class="col-sm-6 control-label"
+             [for]="option.name">
+        {{ option.text }}
+        <br>
+        <span class="text-muted">
+          {{ option.desc }}
+          <cd-helper *ngIf="option.long_desc">
+            {{ option.long_desc }}</cd-helper>
+        </span>
+      </label>
+      <div class="col-sm-6 checkbox-primary checkbox v-center">
```

Comment:
I could not found this class.
```suggestion
      <div class="col-sm-6 checkbox-primary checkbox">
```